docs/getting-started/creating-function.md

```diff
@@ -0,0 +1,60 @@
+# Creating a Knative Function
+
+Knative Functions provide a programming model that leverages the Knative
+Serving and Eventing APIs to provide a quick and easy way for developers
+to deploy their first Knative applications. You don't need to know anything
```

Comment:
Again, this why why I think functions should be after serving and eventing...

Reply:
>  I think functions should be after serving and eventing...

Bottom-up explanations are enticing for those doing the explaining, especailly when one has such broad and deep understanding already, but the words risk falling on deaf ears of eager learners because they have no context upon which to anchor their knowledge.

Instead starting from a context-free starting point, such as the task of deploying a new function which echoes your GET request, or prints "hello world", is easy to grasp.  From this beach-head of knowledge, gradually introducing more advanced topics (serving, eventing, traffic shaping, scaling, etc)  is much more likely to lead to long-term knowledge retention, and consequently adoption.

Reply:
> * Allows you to gradually explore the rest of Knative and Kubernetes when you need it

^^^ this